project-resources/INFRASTRUCTURE_MENU.md

```diff
@@ -46,14 +52,14 @@ The OpenJS Foundation can provide credential storage and sharing through LastPas
 The OpenJS Foundation uses Groups.io for mailing lists on the openjsf.org domain.  All projects are welcome to request their own lists on the @lists.openjsf.org subdomain.
 
 ## Slack
-Projects are welcome to create channels on the OpenJS Foundation Slack (https://openjs-foundation.slack.com), or set up their own free Slack workspace.
+Projects are welcome to create channels on the [OpenJS Foundation Slack](https://openjs-foundation.slack.com), or set up their own free Slack workspace.
 
 ## Zoom
-Projects may request that standing meetings be added to the OpenJS Foundation calendar.  The OpenJS Foundation currently has two Zoom Pro meeting accounts, and one Zoom Webinar account which is capable of livestreaming.  Please be mindful of conflicts with other projects by requesting your meeting be scheduled on the shared calendar via email to operations@openjsf.org.   
+Projects may request that standing meetings be added to the [OpenJS Foundation calendar](https://calendar.openjsf.org).  The OpenJS Foundation currently has multiple Zoom accounts capable of livestreaming.  Please be mindful of conflicts with other projects by requesting your meeting be scheduled on the shared calendar via email to operations@openjsf.org.   
```

Comment:
This feels like it should probably be two separate points - calendar being one, streaming being another.